source/util/AMTimer.h

```diff
@@ -0,0 +1,160 @@
+#ifndef AMTIMER_H
+#define AMTIMER_H
+
+#include <QTimer>
+#include <QTime>
+/*!
+  * A class which provides a wrapper of a QTimer, adding the ability to pause/
+  * unpause, with setPaused(bool) and to find out how long time timer has been
+  * running with elapsedTime().
+  */
+class AMTimer : public QObject
```

Comment:
I worked on the original outline of the class back at the end of May before starting on the SGM Upgrade (Sheldon picked it up off me). To be honest I can't remember the reason I chose one over the other.